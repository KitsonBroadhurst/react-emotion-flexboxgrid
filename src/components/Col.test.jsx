/* eslint-env jest */
/* eslint-disable no-unused-expressions */

import React from 'react';
import { shallow } from 'enzyme';

import Col from './Col';

describe('Col', () => {
  it('should work', () => {
    expect(<Col />).toBeTruthy();
    expect(shallow(<Col />)).toMatchSnapshot();
  });

  it('should be responsive', () => {
    expect(shallow(<Col xs={3} md={4} />)).toMatchSnapshot();
  });
});
