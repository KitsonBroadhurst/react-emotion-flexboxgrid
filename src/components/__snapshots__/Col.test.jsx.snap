// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Col should be responsive 1`] = `
.emotion-0 {
  box-sizing: border-box;
  -webkit-flex: 0 0 auto;
  -ms-flex: 0 0 auto;
  flex: 0 0 auto;
  padding-right: 0.5rem;
  padding-left: 0.5rem;
}

@media only screen and (min-width:0em) {
  .emotion-0 {
    -webkit-flex-basis: 25%;
    -ms-flex-preferred-size: 25%;
    flex-basis: 25%;
    max-width: 25%;
    display: block;
  }
}

@media only screen and (min-width:64em) {
  .emotion-0 {
    -webkit-flex-basis: 33.333333333333336%;
    -ms-flex-preferred-size: 33.333333333333336%;
    flex-basis: 33.333333333333336%;
    max-width: 33.333333333333336%;
    display: block;
  }
}

<div
  className="emotion-0 emotion-1"
/>
`;

exports[`Col should work 1`] = `
.emotion-0 {
  box-sizing: border-box;
  -webkit-flex: 0 0 auto;
  -ms-flex: 0 0 auto;
  flex: 0 0 auto;
  padding-right: 0.5rem;
  padding-left: 0.5rem;
}

<div
  className="emotion-0 emotion-1"
/>
`;
